designer/client/src/list/ListEdit.tsx

```diff
@@ -193,6 +193,44 @@ export function ListEdit() {
           }
           errorMessage={errors.title}
         />
+        <Radios
```

Comment:
Not your fault but the Xgov `<Radios>` component isn't compatible with GOV.UK Frontend v5x

```console
Unhandled Promise Rejection: TypeError: undefined is not a constructor (evaluating 'new RadiosJS(controlRef.current)')
```

Probably one for the backlog, but until we migrate away we can use the radio HTML directly like in:

https://github.com/DEFRA/forms-designer/blob/98fe937bc3d076823c1707273f3830da2167d4ad/designer/client/src/components/FieldEditors/ListContentEdit.tsx#L29-L35